gtests/net/mptcp/sockopts/sockopt_maxseg_tcp.pkt

```diff
@@ -0,0 +1,31 @@
+--tolerance_usecs=100000
+`../common/defaults.sh`
+
++0.0  socket(..., SOCK_STREAM, IPPROTO_TCP) = 3
++0.0  getsockopt(3, SOL_TCP, TCP_MAXSEG, [536], [4]) = 0
+
++0.0  setsockopt(3, SOL_TCP, TCP_MAXSEG, [1000], 4) = 0
++0.0  getsockopt(3, SOL_TCP, TCP_MAXSEG, [1000], [4]) = 0
+
++0.0  bind(3, ..., ...) = 0
++0.0  listen(3, 1) = 0
+
++0.0    <  S   0:0(0)          win 8000  <mss 1460, sackOK, nop, nop, nop, wscale 8>
++0.0    >  S.  0:0(0)  ack 1   win 65535 <mss 1000, nop, nop, sackOK, nop, wscale 8>
++0.01   <   .  1:1(0)  ack 1   win 8000
++0.0  accept(3, ..., ...) = 4
+
++0.0  getsockopt(4, SOL_TCP, TCP_MAXSEG, [1000], [4]) = 0
++0.0  write(4, ..., 2000) = 2000
+
++0.0    >  P.  1:2001(2000)    ack 1
++0.01   <   .  1:1(0)          ack 2001  win 256
+
++0.0  setsockopt(4, SOL_TCP, TCP_MAXSEG, [800], 4) = 0
++0.0  getsockopt(4, SOL_TCP, TCP_MAXSEG, [1000], [4]) = 0
++0.0  write(4, ..., 2000) = 2000
+
++0.0    >  P.  2001:4001(2000) ack 1 win 1055
++0.0    <   .  2001:4001(2000) ack 1 win 1055
```

Comment:
here, I guess that's not what you want: you are sending a packet to ack the sequence 1 (while the previous ACK was acking the sequence 2001, see line 22) with 2000B of data.

Probably you wanted to have this instead?

  +0.01   <   .  1:1(0)          ack 4001  win 256